tests/ffi/common/prepare.sh

```diff
@@ -37,6 +37,18 @@ reload_config() {
 }
 
 prepare_images() {
+   # Update container image_copy_tmp_dir if the image is an OStree.
+   # Default location for storing temporary container image content. Can be overridden with the TMPDIR environment
+   # variable. If you specify "storage", then the location of the
+   # container/storage tmp directory will be used.
+   # By default image_copy_tmp_dir="/var/tmp"
+   # This is a work around and it should not be used constantly
+   if [ -d /run/ostree ]; then
+      exec_cmd "mkdir -p /var/qm/tmp.dir"
+      exec_cmd "mkdir -p /etc/qm/containers/containers.conf.d"
+      exec_cmd "echo -e '[engine]\nimage_copy_tmp_dir=\"/var/tmp.dir\"' > /etc/qm/containers/containers.conf.d/qm_image_tmp_dir.conf"
```

Comment:
> I was not aware of that, https://docs.podman.io/en/latest/markdown/podman.1.html#environment-variables
> 
> thanks
> 
> I think you should call /etc/qm/containers/containers.conf.d/qm_engine_container.conf"
This is what the environment does, once I'm placing this in the end of a config file at /etc/qm/containers/containers.conf.d/qm_image_tmp_dir.conf. I'm posting this at the bottom of the config file, right under the   "[engine]" section and its being addressed by the environment automatically.
image_copy_tmp_dir is the variable that should be changed from its default /var/tmp value to /var/tmp.dir so that the image could get pulled successfully inside the qm.

[engine]
image_copy_tmp_dir="/var/tmp.dir"

> 
> Please add to test README how to update qm container parameters
Will do.

Reply:
Would you like me to create a README as a file, within the folder or just a reference in the code like I did here?
https://github.com/nsednev/qm/blob/a94b15ede6ccd3104f565e9a52132f1d3f49bc0f/tests/ffi/common/prepare.sh#L62C3-L73C15

Reply:
Added README as requested.